content/docs/csidriver/installation/helm/isilon.md

```diff
@@ -96,8 +96,8 @@ You can install CRDs and the default snapshot controller by running the followin
 git clone https://github.com/kubernetes-csi/external-snapshotter/
 cd ./external-snapshotter
 git checkout release-<your-version>
-kubectl create -f client/config/crd
-kubectl create -f deploy/kubernetes/snapshot-controller
+kubectl kustomize client/config/crd | kubectl create -f
```

Comment:
add "-" at the end of command 
ex: kubectl kustomize client/config/crd | kubectl create -f -

Reply:
addressed